api/envoy/extensions/matching/generic_inputs/environment/v3/environment.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+
+package envoy.extensions.matching.generic_inputs.environment.v3;
+
+import "udpa/annotations/migrate.proto";
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.matching.generic_inputs.environment.v3";
+option java_outer_classname = "EnvironmentProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Environment Input]
+// [#extension: envoy.matching.generic_inputs.environment]
+
+// Reads an environment variable to provide an input for matching.
+message Environment {
```

Comment:
`Environment` sounds ambiguous, something more descriptive, perhaps `EnvironmentVariable`, would be clearer.

Reply:
Yea agreed, will update